cmds/bmp/create_baremetals_command_test.go

```diff
@@ -54,29 +57,105 @@ var _ = Describe("create-baremetals command", func() {
 
 	Describe("#Usage", func() {
 		It("returns the usage text of a CreateBaremetalsCommand", func() {
-			Expect(cmd.Usage()).To(Equal("bmp create-baremetals [--dryrun]"))
+			Expect(cmd.Usage()).To(Equal("bmp create-baremetals --deployment[-d] <deployment file> [--dryrun]"))
 		})
 	})
 
 	Describe("#Options", func() {
 		It("returns the options of a CreateBaremetalsCommand", func() {
 			Expect(cmds.EqualOptions(cmd.Options(), options)).To(BeTrue())
+			Expect(cmd.Options().Deployment).ToNot(Equal(""))
+			Expect(cmd.Options().Deployment).To(Equal("../../test_fixtures/bmp/deployment.yml"))
 		})
 	})
 
 	Describe("#Validate", func() {
-		It("validates a good BmsCommand", func() {
+		It("validates a good CreateBaremetalsCommand", func() {
 			validate, err := cmd.Validate()
 			Expect(validate).To(BeTrue())
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		Context("validates a bad CreateBaremetalsCommand", func() {
```

Comment:
Grace, generally, as I mentioned before, the `Context` string starts with "when ..." so something like the following is better:

```
Context("when validating  a bad CreateBaremetalsCommand", func() {...})
```